src/ol_infrastructure/applications/mitopen/__main__.py

```diff
@@ -486,28 +462,7 @@
         ),
     ],
     request_settings=[],
-    snippets=[
-        fastly.ServiceVclSnippetArgs(
-            name="Rewrite requests to root s3",
-            content=textwrap.dedent(
-                rf"""
-                if (req.url.path !~ "{api_regex}") {{
-                   set req.http.orig-req-url = req.url;
-                   set req.url = "/frontend" + req.url;
-                }}
-                if (req.http.retry-for-dir) {{
-                  set req.url = req.url + "/";
-                }}
-
-                # Fetch the index page if the request is for a directory
-                if (req.url ~ "\/$" && req.url !~ "{api_regex}") {{
-                  set req.url = req.url + "index.html";
-                }}
```

Comment:
We still need the logic for implicitly fetching the `index.html` for requests featuring a trailing `/`